packages/metro/src/config/index.ts

```diff
@@ -0,0 +1 @@
+export * from './loadMetroConfig';
```

Comment:
Do we need this file? I think we can either rename `loadMetroConfig` to just `config.js` (no need for `config` folder).

Reply:
There's a 2nd file there: config/metroPlatformResolver.ts. It's used internally by loadMetroConfig.ts.

Reply:
And a lot of types are being exported. Going by a previous comment, I changed * to an explicit list of types/functions being exported for easier maintenance over time.